storage/imx6ull-flash/flashsrv.c

```diff
@@ -743,21 +753,26 @@ static void flashsrv_devThread(void *arg)
 }
 
 
-static int flashsrv_partition(size_t start, size_t size)
+static int flashsrv_addPartition(size_t start, size_t size, const char *name)
 {
 	flashsrv_partition_t *p;
+	int ret;
 
 	p = malloc(sizeof(*p));
 
+	if (p == NULL)
+		return -1;
+
 	p->start = start;
 	p->size = size;
+	p->name = name;
```

Comment:
`strdup()`? You pass to this function `name` placed in non static storage

Reply:
I believe `**argv` is mine to rule inside the program? I can strdup if that would be more readable.

Second thought: if we would ever add partition table support, `strdup` hete might be more straightforward, will fix in a sec